packages/client/src/runtime/dmmf-types.ts

```diff
@@ -1,3 +1,5 @@
 import { DMMF } from '@prisma/generator-helper'
 
 export { DMMF }
+
+export type BundledDMMF = Pick<DMMF.Document, 'datamodel' | 'mappings'>
```

Comment:
I have called that "BaseDMMF", I think it would be good to align/rename all instances to be the same. I'm happy with either "bundled" or "base".

Reply:
`BaseDMMF` sounds good